### ISSUE TITLE
Self Hosting

### ISSUE THREAD

**Issue by MiniDuncan** [dtankdempse/m3u-playlist-proxy#12]
I am using this for daddylive and streamed.su, however I seem to be getting error 429 (I'm being Rate Limited), I'm developing a website I'm hosting locally and it's started rate limiting me as I've been messing around with it. Earlier it was fine. This is only for streamed.su

**Comment by dtankdempse** [comment]
Yes, Streamed Su's token request and the token check, which keeps the token active, are rate limited by IP. If too many requests for a token come from the same IP, they will start blocking them. Ideally, you would need a JavaScript script running on your site to generate the token. This way, each user's own IP address is used when requesting the token."

**Comment by dblviking** [comment]
@dtankdempse first off thank you for your efforts!
Perhaps in your readme, you can note that secure.embedme.top and rr.yayhls.top will have to be whitelisted if you're running any selfhosted Adblock services. Otherwise you'll get a 500 error. 

Looking forward to your self hosted tvapp!
Cheers

**Comment by antebrl** [comment]
@MiniDuncan @dtankdempse At how many concurrent streams are you being rate limited?
